Externals.cfg

```diff
@@ -15,37 +15,37 @@ local_path = components/cice
 required = True
 
 [cime]
-tag = cime5.6.32
+branch = add2300extsspanom
 protocol = git
-repo_url = https://github.com/ESMCI/cime
+repo_url = https://github.com/ekluzek/cime
 local_path = cime
 required = True
 
 [cism]
 tag = cism-release-cesm2.1.2_02
 protocol = git
-repo_url = https://github.com/ESCOMP/cism-wrapper
+repo_url = https://github.com/ESCOMP/CISM-wrapper
 local_path = components/cism
 externals = Externals_CISM.cfg
 required = True
 
 [clm]
-tag = release-clm5.0.30
+branch = rel2300ext
 protocol = git
-repo_url = https://github.com/ESCOMP/ctsm/
+repo_url = https://github.com/ekluzek/CTSM/
```

Comment:
I don't think we want your fork

Reply:
Yes, once there is a cime tag, I can create a CTSM tag on ESCOMP.

Reply:
Got the tag in now.